app/services/users/services.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'backend_services'
+
+module Users
+  class Services
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+      @list = auth_free_services
+    end
+
+    # Checks if the initialized user has authorization to access any of the
+    # below services.  Returns an array of services they have access to.
+    #
+    # @return [Array<String>] Array of names of services they have access to
+    #
+    # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/AbcSize
+    def authorizations
+      @list << BackendServices::RX if user.authorize :mhv_prescriptions, :access?
+      @list << BackendServices::MESSAGING if user.authorize :mhv_messaging, :access?
+      @list << BackendServices::HEALTH_RECORDS if user.authorize :mhv_health_records, :access?
+      @list << BackendServices::MHV_AC if user.authorize :mhv_account_creation, :access?
+      @list << BackendServices::EVSS_CLAIMS if user.authorize :evss, :access?
+      @list << BackendServices::USER_PROFILE if user.can_access_user_profile?
+      @list << BackendServices::APPEALS_STATUS if user.authorize :appeals, :access?
+      @list << BackendServices::SAVE_IN_PROGRESS if user.can_save_partial_forms?
```

Comment:
wasn't the focus of the PR but `can_save_partial_forms?` and `can_access_prefill_data?` are both just hardcoded to return `true`

Reply:
@kreek - I can move those two down into the `#auth_free_services` method, removing the two stubbed out auth checks.  i.e.:

```ruby
    def auth_free_services
      [
        BackendServices::FACILITIES,
        BackendServices::HCA,
        BackendServices::EDUCATION_BENEFITS,
        BackendServices::SAVE_IN_PROGRESS,
        BackendServices::FORM_PREFILL
      ]
    end
```

Or I can keep them as is, I suppose serving as placeholders for future auth logic.  

Thoughts?

Reply:
I think we can move then to free services, the intent of the PR that changed them to `true` was to enable those features for all users https://github.com/department-of-veterans-affairs/vets-api/pull/1254.

The in_progress_forms_controller.rb code will never `raise Common::Exceptions::Unauthorized` and the `prefills_available` method can be simplified.

Reply:
Will do!

Reply:
Completed in https://github.com/department-of-veterans-affairs/vets-api/pull/2738/commits/60880dba09d5e3c3292bdaabaaf2dd9d93fa1771